templates/kubernetes/data-sources/vkcs_kubernetes_cluster.md.tmpl

```diff
@@ -12,6 +12,6 @@ description: |-
 
 ## Example Usage
 
-{{tffile .ExampleFile}}
+{{tffile "examples/kubernetes/cluster/main-datasource.tf"}}
 {{tffile "templates/kubernetes/data-sources/vkcs_kubernetes_cluster/with_id/main.tf"}}
```

Comment:
if a doc page contains several examples each of them must have its own header

Reply:
i left only one